includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Require_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since 1.1.2
+ * @access private
+ * @ignore
+ */
+Trait Require_Javascript_Trait {
+
+	/**
+	 * Display fallback message when Javascript is disabled
+	 *
+	 * @since 1.1.2
```

Comment:
Use this format for all new classes, functions, and methods:
```suggestion
	 * @since n.e.x.t
```

Reply:
Is there a place to reference what the next version will be? Wondering how to know if the next one is a minor or major release.

Reply:
We use `n.e.x.t` as a placeholder which is replaced when preparing the upcoming release, so you don't have to know 😄

Reply:
Ah gotcha, makes sense.